metadata-ingestion/src/datahub/ingestion/source/sql/oracle.py

```diff
@@ -52,13 +57,20 @@ def before_cursor_execute(conn, cursor, statement, parameters, context, executem
     cursor.outputtypehandler = output_type_handler
 
 
+def class_usage_notification(cls, func):
```

Comment:
This can be removed, right ? Its not used anywhere.

Reply:
Yes, it can. Main idea behind this method consist in logging usage of wrapper class.